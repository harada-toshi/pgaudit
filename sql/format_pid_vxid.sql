\! echo "pgaudit.config_file = '/tmp/config.conf'" >> ./tmp_check/data/postgresql.conf
\! rm ./tmp_check/tmp_check.log

\! cp sql/format_pid_vxid/format_pid_vxid.conf /tmp/config.conf
\! pg_ctl restart -w -D ./tmp_check/data -l ./tmp_check/tmp_check.log
\! psql contrib_regression -q -a < sql/format_pid_vxid/format_pid_vxid.sql
\! rm /tmp/config.conf

\! grep "AUDIT:" ./tmp_check/tmp_check.log | sed 's/[0-9]\+/X/g'> results/format_pid_vxid.out

