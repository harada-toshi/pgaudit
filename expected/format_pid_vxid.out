	        format = 'AUDIT:(A) %pid,%virtual_xid,%command_text'
	        format = 'AUDIT:(B) %p,%v,%command_text'
LOG:  AUDIT:(A) X,X/X,SELECT 'ABC';
LOG:  AUDIT:(B) X,X/X,SELECT 'ABC';
LOG:  AUDIT:(A) X,X/X,SELECT * FROM foo;
LOG:  AUDIT:(B) X,X/X,SELECT * FROM foo;
